Tests/NodesGeneratorTests/Support/TestFactories.swift

```diff
@@ -218,6 +220,7 @@ extension TestFactories {
             flowTestsImports: .mock(with: "flowTestsImport", count: mockCount),
             interfaceImports: .mock(with: "interfaceImport", count: mockCount),
             pluginImports: .mock(with: "pluginImport", count: mockCount),
+            pluginInterfaceImports: .mock(with: "pluginInterfaceImports", count: mockCount),
```

Comment:
This should be singular.
```suggestion
            pluginInterfaceImports: .mock(with: "pluginInterfaceImport", count: mockCount),
```